### PR TITLE
Adopt Laminas CI workflow

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # laminas-cache-storage-adapter-apcu
 
-[![Build Status](https://travis-ci.com/laminas/laminas-cache-storage-adapter-apcu.svg?branch=master)](https://travis-ci.com/laminas/laminas-cache-storage-adapter-apcu)
-[![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-cache-storage-adapter-apcu/badge.svg?branch=master)](https://coveralls.io/github/laminas/laminas-cache-storage-adapter-apcu?branch=master)
+[![Build Status](https://github.com/laminas/laminas-cache-storage-adapter-apcu/workflows/continuous-integration.yml/badge.svg)](https://github.com/laminas/laminas-cache-storage-adapter-apcu/actions/workflows/continuous-integration.yml)
```
Comment (froschdesign) [inline README.md]:
```suggestion
[![Build Status](https://github.com/laminas/laminas-cache-storage-adapter-apcu/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/laminas/laminas-cache-storage-adapter-apcu/actions/workflows/continuous-integration.yml)
```

**Comment by boesing** [comment]
Before adding more PRs like these, we have to fix the circular dependency problem as in https://github.com/laminas/laminas-cache-storage-adapter-apc/pull/4

**Comment by Slamdunk** [comment]
May I ask you why this PR have to wait for https://github.com/laminas/laminas-cache-storage-adapter-apc/pull/4 ?
It seems to me that the CI is running good, and whatever we need to fix there, it would be better to do it **with** this PR already in place.
I'm sorry @boesing I'm trying my best to help with `laminas-cache`, I fail to understand how a passing PR should be blocked by something completely dissociated; it really halts my willingness to help as it seems there's a lot *todo*, but no one else except you knows what the *todo* is

**Comment by Slamdunk** [comment]
Since I believe what I'm doing here is more good than harm, I'll continue opening this PR for all other components nevertheless.
I can put my effort now, and I don't want to waste it.

**Comment by boesing** [comment]
It is absolutely good, there is no doubt about that.

The problem is, that we still have to create a PR **again** in all adapters to fix the problem. 
So if you continue creating PRs in all adapters without that fix, we are wasting time (even if thats not yours) in creating multiple PRs which have to get reviewed and merged.

**Pull request merged**